components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.popup.js

```diff
@@ -166,7 +166,7 @@ String.prototype.escapeHTML = function(){
     return s.replace(/[\&"<>\\]/g, function(s) {
         switch(s) {
             case "&": return "&amp;";
-            case "\\": return "\\\\";
+            // case "\\": return "\\\\";
```

Comment:
This line doesn't escape single backslash, there must be something else in the code